### PR TITLE
Fixed rime-tsch sync problem in Cooja

### PR DESCRIPTION
**Pull request by e120guru**
The rime-tsch example does not sync properly in cooja when using cooja motes. Compiling for instance a sky mote works.

As it turns out there seems to be lacking the directive to simulate the turnaround time in this example.

This PR adds a fix for this, after applying this fix cooja motes syncronize as expected even when using cooja motes.

### CODE REVIEW

**Comment by yatch** [comment]
You should be right. `COOJA_CONF_SIMULATE_TURNAROUND` needs to be set with 0 when you're using TSCH witt Cooja mote as I mentioned in the Introduction section of https://github.com/contiki-os/contiki/pull/1661.

Sorry for the inconvenience. I should have added something about `COOJA_CONF_SIMULATE_TURNAROUND` into README of TSCH.  I'll send a PR to update the README.

File: examples/rime-tsch/project-conf.h [hunk 1]
```diff
@@ -85,4 +85,10 @@
 #undef TSCH_LOG_CONF_ID_FROM_LINKADDR
 #define TSCH_LOG_CONF_ID_FROM_LINKADDR(addr) ((addr) ? (addr)->u8[LINKADDR_SIZE - 2] : 0)
 
+#if CONTIKI_TARGET_COOJA
+#define COOJA_CONF_SIMULATE_TURNAROUND 0
+#define COOJA_CONF_TRANSMIT_ON_CCA 1
```
Comment (yatch) [inline examples/rime-tsch/project-conf.h]: This one is needed as well to run TSCH with Cooja...? As per `platform/cooja/dev/cooja-radio.h`, the default value of `COOJA_TRANSMIT_ON_CCA` is `1`... I may miss something.

**Comment by simonduq** [comment]
Right, would be a good idea to centralize the TSCH-related configs somewhere, at least in the readme! Note that we have some both at the beginning and the end of the project-conf examples..

**Comment by simonduq** [comment]
I agree the def of `COOJA_CONF_TRANSMIT_ON_CCA` does not seem needed. Other than that, thanks for the catch!

**Comment by e120guru** [comment]
PR updated, removed redundant line

**Comment by simonduq** [comment]
thanks. if you could just squash the two commits into one, this is ready to merge!

**Comment by e120guru** [comment]
PR updated

**Comment by simonduq** [comment]
Thanks! 👍

**Pull request merged**